### DISCUSSION TITLE
Can not write the new geometry of my lighthouse setup to the drones

### DISCUSSION THREAD

**Discussion by bo3lwa1** [bitcraze/discussions#1210] [Q&A]
Am working with the LH swarm bundle. I will demonstrate my issue in steps to deliver the message clearly:
I connected the first drone, set up the LH, estimated the geometry, and wrote the geometry to the drone. Then, I did some tests (take off, right, and left) everything is working perfectly. I disconnect the first drone and connect the second one. I loaded the same geometry that I saved but the write to crtazyflie button is off so I couldn't write the geometry to it. when I did some takeoff tests it crushed directly.
I think the problem is I didn't write the new geometry for the drone. how can I write the geometry of the already set LH to the drone? Is there a way to write the geometry to all drones at once?
In addition, I tried the takeoff tests in cfclient and skubrush live, and it gave me the same result.

**Comment by knmcguire** [comment]
Hi!

So if I understood correctly, 
1- you managed to save a .yaml file from the first crazyflie of the geometry that you estimated (which flew fine) in [the system management](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/lighthouse_tab/#system-management)
2- When you connect the cfclient to the 2nd crazyflie, the 'write to crazyflie' button in the [basestation geometry management](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/lighthouse_tab/#basestation-geometry-management) is grayed out

If the geometry is not uploaded on the Crazyflie than it doesn't make sense to try it to fly, since the position estimate will be completely off. But... the normal way of handling it is that , the [load system config button](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/lighthouse_tab/#system-management) should have already been enough to load the new geometry estimate to the crazyflie, and the reason why the button was gray was that there was nothing new to write to the crazyflie.

Some followup questions:
- Which version of the crazyflie-firmware does the 2nd crazyflie have?
- What does the [console tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/console_tab/) of the 2nd not working crazyflie say?
- Are you able to see the basestation location on the lighthouse tab's visualization
- What is the position estimate doing? You can check that either on the LH tab viz or [the plotter tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/plotter_tab/)

**Reply by bo3lwa1** [reply]
Our second drone has 2023.11 firmware.
this a screenshot of our Consol tab
![wd](https://github.com/bitcraze/discussions/assets/161849297/b0b7d95b-fddf-4e97-b07e-dfda3fa5c30c)


Yes, LH base station locations were shown in the LH tab.

Apparently,  after connecting the 2nd drone to the cfclient and then clicking "load system config" button, we had to just turn the drone off then on again for the drone to fly perfectly and have the configuration.

Thank you for the quick responding,

**Reply by knmcguire** [reply]
ah oke! bit weird since it should be necessarsy to restart the crazyflie but it all works now in the end. Happy that you are able to continue now.
